pcsx2/DEV9/AdapterUtils.cpp

```diff
@@ -244,30 +247,70 @@ std::optional<MAC_Address> AdapterUtils::GetAdapterMAC(Adapter* adapter)
 
 	return std::nullopt;
 }
-#elif defined(__POSIX__)
-#ifdef __linux__
+#else
 std::optional<MAC_Address> AdapterUtils::GetAdapterMAC(Adapter* adapter)
 {
-	struct ifreq ifr;
-	strcpy(ifr.ifr_name, adapter->ifa_name);
+	MAC_Address macAddr = {};
+#if defined(AF_LINK)
+	ifaddrs *adapterInfo;
```

Comment:
```suggestion
	ifaddrs* adapterInfo;
```

Our style has the asterisk on the type, not the variable. If you clang-format this section, it should apply this anyway.

Reply:
Sorry I forgot to apply format style, done!